doc/kubevela-definitions/cardano/cardano-wallet-connector.md

```diff
@@ -0,0 +1,39 @@
+# Cardano Wallet Connector trait
+
+## Overview
+
+The Cardano Wallet Connector trait in KubeVela serves as a pass-through between your application and the Cardano wallet, using specific application characteristics and properties.
+
+The ***required*** parameter for implementing this trait is:
+ - `network`: Defines the network choice for the Cardano Wallet. The available options are "preview", "preprod" and "mainnet".
+
+This configuration effectively attaches specific volumes to each container that is part of your application. Also, a `socat` container bridges the Cardano Wallet and the Cardano Node by utilizing TCP Socket over a UNIX Socket.
```

Comment:
These implementation details don't belong in the "overview", and probably not in the user docs at all. Can probably just say something like "This configuration makes a cardano-wallet instance available for your application"